src/content/guides/author-libraries.md

```diff
@@ -99,15 +100,15 @@ import * as webpackNumbers from 'webpack-numbers';
 webpackNumbers.wordToNum('Two');
 ```
 
-- __CommonJS module require:__
+- __CommonJS module requires:__
```

Comment:
that is a header for example require call of `CommonJS ` module, why is this change necessary?